### ISSUE TITLE
Updates / project status?

### ISSUE THREAD

**Issue by jamesvl** [handnot2/esaml#33]
@handnot2 I've noticed merge requests have been sitting open a for a while, and with OTP 24 out now deprecated some functions used here, are you still interesting in maintaining this project moving forward? If not, would you consider letting others help out in some fashion?

I know this work has been hugely valuable for me to do some SSO integrations I would not have been able to do (via samly) otherwise, and am willing to pitch in to make sure both remain viable going foward.

**Comment by tcrossland** [comment]
@jamesvl agreed, I think it would be useful for the community if this project were more actively maintained. I believe the BSD license allows the work to be modified and would be supportive of moving the maintenance effort to a fork if this repository is no longer active.

@handnot2, @arekinath many thanks for all your work on esaml. Please respond if you would rather continue to use this repo for maintaining esaml.

**Comment by arekinath** [comment]
I'm happy to create a "esaml" org account and move the original repo there, and give @handnot2 admin on the org (and commit access to others who are maintaining it), if that seems useful?

**Comment by jamesvl** [comment]
I'd certainly be willing to help triage or do reviews if we pursued this path.

Having an official move of the repo would be nice for anyone using the hex packages; it would be ideal to avoid creating a new hex project if we could.

**Comment by tcrossland** [comment]
@arekinath the esaml org account seems like a good idea to me... the only issue is that @handnot2 doesn't seem to be responding and has no contribution activity since april 2019, so it might be a good idea to have some other admins from the community (@jamesvl or myself, for example).

What do you think?

**Comment by k-cross** [comment]
I have forked this to maintain it through dropbox here: https://github.com/dropbox/esaml

I'm currently in the process of getting ownership of the project through hex.pm too. Are there any other places that I should try to take over ownership? I've been handling all of the corporate takeover stuff since October.

**Comment by mitchwd** [comment]
Hey @k-cross,
Looks like you've published some new releases since Nov 2021, but the repo at https://github.com/dropbox/esaml is hidden from the public. Any ideas? Is your fork still being maintained?

**Comment by k-cross** [comment]
@mitchwd yeah, somehow it was removed. i went through the same convo with someone else. it's not maintained by me anymore and really should move to its own org. dbx has not been a good steward of the project and likely don't care. i also no longer work there and have no authority. if i have hex credentials to pass off, i'm more than happy to give them to people who want to push the project forward.

**Comment by mitchwd** [comment]
Thanks for the update @k-cross. Disappointed to hear that Dropbox has dropped all your hard work. Thanks for your efforts, we'll try something else.

**Comment by jamesvl** [comment]
My company recently needed SAML again and we maintain a fork of `samly` at https://github.com/Semsee/samly 

I haven't had to do as much with `esaml` but if it needs updates I have a better shot now with AI assisting. My company can volunteer to help steward this along if there's interest.

**Comment by k-cross** [comment]
It's better to move this into a separate org @jamesvl that way anyone wanting to take ownership of the project simply needs to pass the baton instead of redirecting all the users to another site each time stwardship changes.

**Comment by jamesvl** [comment]
I'm fine with that - want me to set up a new org and get this moving?

**Comment by k-cross** [comment]
Yeah, I think that's a great idea!

**Comment by jamesvl** [comment]
Lacking any other creative name (and making room in case I add other projects that need a home), I've created a new org at https://github.com/an-elixir-community and invited k-cross.

I'll get samly and esaml added there in short order.
